src/database/mongooseSchema.js

```diff
@@ -78,7 +78,7 @@ const User = mongoose.model(
 const Notification = mongoose.model(
 	'Notification',
 	new Schema({
-		article: { type: mongoose.Schema.Types.ObjectId, ref: 'Article', required: true, unique: true },
```

Comment:
@shrijan-siris we need a composite key for article & user.
e.g. https://stackoverflow.com/questions/16061744/mongoose-how-to-define-a-combination-of-fields-to-be-unique

Reply:
Got it, fixing this

Reply:
@syuraj  i am closing this pr as i am creating new one for that